source/default/_patterns/00-protons/non-printing/_bootstrap-overrides.scss

```diff
@@ -16,6 +16,12 @@ $btn-border-radius: 0.25rem;
 // Cards
 $card-border-width: rem-calc(1px);
 
+// Spacing
+$spacers: (
+  6: (1rem * 3.5),
```

Comment:
Use font-base-var here.

Reply:
Hey @illepic, I tried replacing `1rem` with `$font-size-base` from `_variables.scss` but got the error `"Undefined variable: "$font-size-base"`. I think it's because the vars are imported after the overrides. I also tried adding the overrides after the @imports but then only the overrides were rendered to the page.